libs/components/colorpicker/src/lib/modules/colorpicker/colorpicker.component.scss

```diff
@@ -11,34 +100,146 @@ sky-colorpicker.sky-margin-stacked-lg {
-  border: 0;
+  // the padding between the inner and outer borders should be 2px. Since the borders are stacked we need to add the size of the previous border to the intended size.
+  --sky-colorpicker-border-padding-size: calc(
+    var(--sky-border-width-input-base) +
+      calc(var(--sky-border-width-input-base) * 2)
+  );
+  --sky-color-picker-inner-border-size: calc(
+    var(--sky-colorpicker-border-padding-size) +
+      var(--sky-border-width-input-base)
+  );
+  height: var(
+    --sky-override-colorpicker-button-size,
+    calc(
+      calc(var(--sky-font-line_height-body-m) * var(--sky-font-size-body-m)) +
+        calc(var(--sky-space-inset-balanced-s) * 2)
+    )
+  );
+  width: var(
+    --sky-override-colorpicker-button-size,
+    calc(
+      calc(var(--sky-font-line_height-body-m) * var(--sky-font-size-body-m)) +
+        calc(var(--sky-space-inset-balanced-s) * 2)
+    )
+  );
+  border: none;
+  border-radius: var(
+    --sky-override-colorpicker-button-border-radius,
+    var(--sky-border-radius-s)
+  );
+  box-shadow: var(
+    --sky-override-colorpicker-button-box-shadow,
+    inset 0 0 0 var(--sky-border-width-input-base)
+      var(
+        --sky-override-colorpicker-button-border-color,
+        var(--sky-color-border-input-base)
+      ),
+    inset 0 0 0 var(--sky-colorpicker-border-padding-size)
+      var(--sky-color-background-container-base),
+    inset 0 0 0 var(--sky-color-picker-inner-border-size)
+      var(
+        --sky-override-colorpicker-button-border-color,
+        var(--sky-color-border-input-base)
+      )
+  );
   cursor: pointer;
+  position: relative;
+
+  &:hover {
+    box-shadow: var(
+      --sky-override-colorpicker-button-box-shadow,
+      inset 0 0 0 var(--sky-border-width-input-hover)
+        var(--sky-color-border-input-hover),
+      inset 0 0 0 var(--sky-colorpicker-border-padding-size)
+        var(--sky-color-background-container-base),
+      inset 0 0 0 var(--sky-color-picker-inner-border-size)
+        var(
+          --sky-override-colorpicker-button-border-color,
+          var(--sky-color-border-input-base)
+        )
+    );
+  }
+
+  &:focus-visible:not(:active) {
+    box-shadow: var(
+      --sky-override-colorpicker-button-box-shadow,
+      inset 0 0 0 var(--sky-border-width-input-focus)
+        var(--sky-color-border-input-focus),
+      inset 0 0 0 var(--sky-colorpicker-border-padding-size)
+        var(--sky-color-background-container-base),
+      inset 0 0 0 var(--sky-color-picker-inner-border-size)
+        var(
+          --sky-override-colorpicker-button-border-color,
+          var(--sky-color-border-input-base)
+        ),
+      var(--sky-elevation-focus)
+    );
+    outline: var(--sky-override-colorpicker-button-focus-outline);
+  }
+
   &.sky-colorpicker-button-disabled {
     cursor: default;
+    background-color: var(
+      --sky-override-colorpicker-disabled-background,
+      var(--sky-color-background-input-disabled)
+    ) !important;
```

Comment:
This `!important` already existed, it is necessary because we set the background color to the selected color via inline styles